Automattic/wp-calypso
# Add trial Plans page

#### Proposed Changes

* This PR adds the custom Plans page for the eCommerce trial Plan.
* **It's still missing the final copy and actions for the buttons/links.**
* **Monthly/Annually toggle is gonna be added in a future PR**

#### Testing Instructions
* Open the "Plan" page (`/plans/<site-slug>`) for a site with the eCommerce trial plan
* Check that the new design is shown and that it matches the Figma design 

Figma:
![POx7l2.png](https://user-images.githubusercontent.com/3801502/217605585-97576f8a-ba05-493d-9fd3-5bba281c60af.png)

HTML:
![k8srIG.png](https://user-images.githubusercontent.com/3801502/217605304-461709c6-5f72-45dd-94b3-178e3402566d.png)

* Test accordion functionality.

* Open in mobile mode and compare with the designs:

Figma:
![4RqCGb.png](https://user-images.githubusercontent.com/3801502/217605688-4ca7376f-d37b-41df-944f-701c5c3de5fe.png)

HTML:
![Vt4fxY.png](https://user-images.githubusercontent.com/3801502/217608817-58b4e6a2-2ff4-4e0d-8fb5-4a181db46a99.png)



Closes https://github.com/Automattic/wp-calypso/issues/71807
